guide/03-the-gis/cloning-content.ipynb

```diff
@@ -0,0 +1,2051 @@
+{
```

Comment:
<p>In paragraph 1, maybe to rewrite "The Python API team concluded we needed a guide for Cloning Content. We tackled the task to make the ultimate roadmap for how to get content from one Web GIS to another." from user's perspective?</p><p> </p><p>Then in paragraph 3, "Rather than tangle ourselves up when trying to distinguish between these portal types while writing about them, we'll refer to the portal or organization where the content originates as the <em>source</em> regardless of portal type. " - *tangling*, *ArcGIS organization*</p><p> </p><p>Also, in paragraph 4, "<code>Web Maps</code> consume these data service as. operational layer building blocks." - please remove the period.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/677/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<ul><li>I edited the intro paragraph to read from the user perspective. How does this sound?:</li></ul><p> </p><p>				<em>One request the Python API team has heard repeatedly from Web GIS administrators: What's the best way </em></p><p>				<em>to move my content from a development Enterprise or organization to stagint to production?</em></p><p> </p><p>				<em>This Guide provides one possible roadmap for how to get content from one Web GIS to another - a template </em></p><p>				<em>for demonstrating basic software concepts when transferring content so the workflow can be modified for </em></p><p>				<em>other types of content and tailored for an administrator's particular need. So, first question to answer: What </em></p><p>				<em>does it mean to</em><code><em>clone</em></code><em>?.....</em></p><p> </p><ul><li>fixed the <code>tangling</code>  typo</li><li>remved extra period</li></ul>

Reply:
<p>Typo at "Cloning can refer to all kinds of transfer, from ArcGIS Enterprise to ArcGIS Online, Enterprise to Enterprise, or<strong> ArCGIS</strong> Online to Enterprise"</p>